hack/lib/golang.sh

```diff
@@ -162,6 +163,7 @@ readonly KUBE_STATIC_LIBRARIES=(
   kube-scheduler
   kube-proxy
   kubectl
+  federated-apiserver
```

Comment:
FYI, I am renaming this to federation-apiserver in https://github.com/kubernetes/kubernetes/pull/26196